scriptspony/templates/request_cert.mak

```diff
@@ -0,0 +1,66 @@
+<%inherit file="scripts.templates.master"/>
+
+<%!
+from scripts.auth import token
+
+
+%>
+
+<div><a href="${tg.url('/')}">&laquo; Back to list</a></div>
+
+<h3> The TLS Certificate Process</h3>
+<p>
+    The process to get a TLS certificate in order to use HTTPS on your scripts domain is currently as follows: 
+</p>
+<ul>
+    <li> You generate a certificate-signing request (CSR) for your host by filling out the form below.</li>
+    <li> You get a certificate from a certificate authority (CA) using this CSR.</li>
+    <li> Once you have a certificate, paste it below and submit the second form.</li>
+</ul>
+
+
+%if show_only_csr:
+<h3> CSR Request Results </h3>
+
+    % if csr_success:
+        <p> Your CSR was successfully generated and appears below.</p>
+    % else:
+        <p> The following error was generated. Please contact us for help.</p>
+    % endif
+    <pre>${csr_contents}</pre>
+%else:
+
+<h3> Request a CSR</h3>
+<p>
+Choose which aliases to include:<br><ul>
```

Comment:
I fixed that and a bunch of other things. The validator is still complaining about the `form` and `placeholder` attributes on my textarea. What's the rationale for not allowing HTML5 features that won't detract from older browsers' experiences?